core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairTask.java

```diff
@@ -211,30 +209,27 @@ public String toString()
         options.put(RepairOptions.PARALLELISM_KEY, myRepairConfiguration.getRepairParallelism().getName());
         options.put(RepairOptions.PRIMARY_RANGE_KEY, Boolean.toString(false));
         options.put(RepairOptions.COLUMNFAMILIES_KEY, myTableReference.getTable());
-        options.put(RepairOptions.INCREMENTAL_KEY, Boolean.toString(myRepairConfiguration.getRepairType().equals(RepairOptions.RepairType.INCREMENTAL)));
+        options.put(RepairOptions.INCREMENTAL_KEY, Boolean.toString(false));
 
-        if (vnodeRepair)
-        {
-            StringBuilder sb = new StringBuilder();
-
-            for (LongTokenRange range : myTokenRanges)
-            {
-                sb.append(range.start).append(':').append(range.end).append(',');
-            }
+        StringBuilder rangesStringBuilder = new StringBuilder();
 
-            options.put(RepairOptions.RANGES_KEY, sb.toString());
+        for (LongTokenRange range : myTokenRanges)
+        {
+            rangesStringBuilder.append(range.start).append(':').append(range.end).append(',');
         }
 
+        options.put(RepairOptions.RANGES_KEY, rangesStringBuilder.toString());
+
         if (myReplicas != null)
         {
-            StringBuilder sb = new StringBuilder();
+            StringBuilder replicasStringBuilder = new StringBuilder();
 
             for (Host host : myReplicas)
             {
-                sb.append(host.getBroadcastAddress().getHostAddress()).append(',');
+                replicasStringBuilder.append(host.getBroadcastAddress().getHostAddress()).append(',');
```

Comment:
Maybe it does not matter that it is a "," after the last address in the list?
But may still be nice to use a stream/collect/joining to create the string of addresses... Something like this:
String hostAddresses = myReplicas.stream()
    .map(host -> host.getBroadcastAddress().getHostAddress())
    .collect(joining(","));

Reply:
Looking in the Cassandra code it's parsed with a [StringTokenizer](https://github.com/apache/cassandra/blob/cassandra-3.0/src/java/org/apache/cassandra/repair/messages/RepairOption.java#L189). The StringTokenizer handles the last ",".

But I think your suggestion is correct so I'll change the implementation accordingly.